topiary/tests/samples/input/ocaml.ml

```diff
@@ -35,11 +35,11 @@ let id1 = if true then Fun.id else function x -> x
 
 let id2 = if true then Fun.id else function x -> x | y -> y
 
-let id3 = if 
-true 
-then 
-Fun.id 
-else 
+let id3 = if
+true
+then
+Fun.id
+else
```

Comment:
Input files should probably be excluded from the pre-commit stuff. Or at least, we should have a way to test if Topiary correctly removes trailing whitespace.